R/testdata.R

```diff
@@ -208,87 +240,77 @@ dataset_test_worker <-
 
       is_allowed <- i %in% charToRaw(exceptions)
       !(is_ascii | is_allowed)
-
     }
 
     # Better than expect_silent as contains `info` and allows for complete failures
-    expect_no_error <-
-      function(object, regexp = NULL, ..., info = NULL, label = NULL) {
-        error <- tryCatch({
-          object
-          NULL
-        }, error = function(e) {
-          e
-        })
-        expect(is.null(error),
-               sprintf(
-                 "%s threw an error: %s",
-                 label,
-                 paste(error$message, collapse = ",")
-               ),
-               info = info)
-        invisible(NULL)
-      }
+    expect_no_error <- function(object, regexp = NULL, ..., info) {
```

Comment:
Looks like regex isn't being used?

Reply:
Yeah it's not being used, I didn't add it so happy to get rid of it.